scannerazure.properties

```diff
@@ -6,8 +6,13 @@ organizationUrl=https://www.sonarsource.com
 homepageUrl=https://docs.sonarqube.org/latest/analysis/scan/sonarscanner-for-azure-devops/
 issueTrackerUrl=https://jira.sonarsource.com/browse/VSTS/issues
 sourcesUrl=https://github.com/SonarSource/sonar-scanner-vsts
-archivedVersions=4.7,4.7.1,4.7.2,4.8,4.8.1,4.9,4.10
-publicVersions=4.11
+archivedVersions=4.7,4.7.1,4.7.2,4.8,4.8.1,4.9,4.10,4.11
+publicVersions=4.12
+
+4.12.description=Support for .NET 5, support for solo .NET Core project (without .sln)
+4.12.date=2020-11-05
+4.12.changelogUrl=https://jira.sonarsource.com/secure/ReleaseNote.jspa?projectId=10986&version=16333
```

Comment:
Doesn't seem to be the correct changelogUrl link

Reply:
it was also the same for 4.11 and 4.10, I don't know if there is a proper changelog url

Reply:
the `version` differs at the end:
4.12 should be https://jira.sonarsource.com/secure/ReleaseNote.jspa?projectId=10986&version=16488